src/pyspiffe/workloadapi/x509_context.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+
+from pyspiffe.bundle.x509_bundle.x509_bundle_set import X509BundleSet
+from pyspiffe.svid.x509_svid import X509Svid
+
+__all__ = ['X509Context']
+
+
+class X509Context(object):
```

Comment:
Aren't there tests for this class?

Reply:
I hadn't added tests because it's just like a DTO object without much logic, but yeah it has some cases that should be tested.